src/webui/www/private/preferences_content.html

```diff
@@ -660,7 +660,6 @@
     updateForceProxySettings = function() {
         var isForceProxyEnabled = (!$('force_proxy_checkbox').getProperty('disabled')) && ($('force_proxy_checkbox').getProperty('checked'));
         $('upnp_checkbox').setProperty('disabled', isForceProxyEnabled);
-        $('dht_checkbox').setProperty('disabled', isForceProxyEnabled);
```

Comment:
Do GUI follow this behavior already? If not, do you mind also apply the change to GUI? (And make it a separate commit)

Reply:
The GUI already has this behavior, though I actually plan to remove it from both per https://github.com/arvidn/libtorrent/pull/3251.

Reply:
Seems not? https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/optionsdialog.cpp#L1361

Reply:
Sorry, I don’t follow. Are you saying the GUI doesn’t currently disable the DHT checkbox when force proxy is enabled?

Reply:
Ahh, my confusion was because I didn't realize I had added that change to this PR. I'll remove it and open a separate PR for that change to the WebUI and GUI.